build.gradle

```diff
@@ -378,12 +404,10 @@ dependencies {
      * https://tomcat.apache.org/tomcat-8.5-doc/RELEASE-NOTES.txt
      */
     provided 'javax.annotation:javax.annotation-api:1.3.2'
-    provided 'javax.el:javax.el-api:2.2.5'
```

Comment:
this was a minor mistake in OBGM-363, we don't particularly care what el-api release we use...